test-suite/Makefile.am

```diff
@@ -140,9 +141,25 @@ VirtualDeleteOperator_SOURCES = VirtualDeleteOperator.cc stub_libmem.cc $(DEBUG_
 ##$(TARGLIB): $(LIBOBJS)
 ##	$(AR_R) $(TARGLIB) $(LIBOBJS)
 
-squid-conf-tests: $(top_builddir)/src/squid.conf.default $(srcdir)/squidconf/*
+tls-test-certs: $(srcdir)/mk-tls-test-certs.sh
+	$(srcdir)/mk-tls-test-certs.sh
```

Comment:
This script should not be run for a Squid ./configured without TLS support.

Similarly, the new `tls-certs` configuration file should not be included in `SQUID_CONFIGS` for a Squid ./configured without TLS support. I have no good source code place to attach this paragraph to because `SQUID_CONFIGS` (or a similar automake variable) does not exist yet -- another change request introduces `SQUID_CONFIGS`.